diff --git a/dwm.c b/dwm.c
--- a/dwm.c	2020-07-30 13:25:38.717684932 +0200
+++ b/dwm.c	2020-07-30 13:25:38.717684932 +0200
@@ -685,6 +685,9 @@
 			updatesystrayicongeom(c, wa.width, wa.height);
 			XAddToSaveSet(dpy, c->win);
 			XSelectInput(dpy, c->win, StructureNotifyMask | PropertyChangeMask | ResizeRedirectMask);
+			/* give systray a WM_CLASS  */
+			XClassHint ch = {"dwmsystray", "dwmsystray"};
+			XSetClassHint(dpy, c->win, &ch);
 			XReparentWindow(dpy, c->win, systray->win, 0, 0);
 			/* use parents background color */
 			swa.background_pixel  = scheme[SchemeNorm][ColBg].pixel;
